release_build_files/readme.md

```diff
@@ -627,6 +627,11 @@ workflow use only during the development of your app, not for publicly shipping
 code.
 
 ## Release Notes
+### Upcoming Release
+-   Changes
+    - Auth (Android): Fixed an issue where VerifyPhoneNumber's internal
```

Comment:
How would this actually occur to a developer using our SDK? Would it always fail or only if they specified a timeout? Probably best to describe this from the developer's perspective.

Reply:
The issue was reported by a developer in https://github.com/firebase/firebase-unity-sdk/issues/764.
I tried to phrase the readme description similar to the error message that was shown to the user.

I think it would always fail. VerifyPhoneNumber takes a PhoneAuthOptions which always has a timeout milliseconds field. Regardless of what they set (0 or 60 seconds), it would still overflow to some invalid value and the user gets an error.

That said, when I tested our C++ integration tests VerifyPhoneNumber succeeded even without this fix. I think this must be something like a different compiler that uses 32 bits for a long, and therefore no overflow occured.

Reply:
Ah ok. So maybe add "with certain compiler settings" to the end of the explanation.

Reply:
Done